test/default-xpring-client-test.ts

```diff
@@ -401,4 +401,34 @@ describe('Default Xpring Client', function(): void {
       done()
     })
   })
+
+  it('Check if account exists - exists with valid address and positive balance', async function() {
```

Comment:
I'd retitle these as what we actually care about: "- successful network request" and "- failing network request". Open to suggestions if you think this title is conveying useful information.